docs/source/frameworks/keras.rst

```diff
@@ -2,50 +2,142 @@
 Keras
 =====
 
-Users can now use Keras (with Tensorflow v2 backend) with BentoML with the following
-APIs:
+.. note::
 
-.. code:: python
+   Both ``bentoml.keras`` and ``bentoml.tensorflow`` support Keras
+   model. ``bentoml.keras`` utilizing Keras' native model format,
+   which will give better development experience if user are more
+   familiar with Keras model. However, Keras' native model format is
+   not optimized for production inference. Using
+   ``bentoml.tensorflow`` is recommended in production
+   environment. You can read :doc:`bentoml.tensorflow documentation
+   </frameworks/tensorflow>` for more information.
```

Comment:
Maybe for this part, bring the part in bentoml.tensorflow docs from jiangbo PR to here, and explain some of the performance deficit about using Keras.